docs/manage/task-config-policies.rst

```diff
@@ -0,0 +1,157 @@
+.. _task-config-policies:
```

Comment:
should this filename/path include `task`?

Reply:
resolved